src/modules/dashboard/components/ActionsPage/ActionsComponents/MintTokenMotion.tsx

```diff
@@ -165,6 +166,7 @@ const MintTokenMotion = ({
           <MemberReputation
             walletAddress={actionInitiator}
             colonyAddress={colony.colonyAddress}
+            rootHash={rootHash || undefined}
```

Comment:
Because of the types: rootHash is either string or null. So I either pass undefined here or I need to add extra null type for rootHash in other components. I went for the above option.